src/VisualStudio/IntegrationTests/SharedIntegrationHostFixture.cs

```diff
@@ -8,6 +9,11 @@ namespace Roslyn.VisualStudio.IntegrationTests
     [CollectionDefinition(nameof(SharedIntegrationHostFixture))]
     public sealed class SharedIntegrationHostFixture : ICollectionFixture<VisualStudioInstanceFactory>
     {
+        public const string RoslynLanguageServicesPackageId = "0b5e8ddb-f12d-4131-a71d-77acc26a798f";
```

Comment:
This will actually need to change, the packages aren't the VSIX ID (like I assumed) it is the VS Authoring name